include/vecmath/scalar.h

```diff
@@ -589,7 +589,7 @@ namespace vm {
     template <typename T>
     constexpr bool is_zero(const T v, const T epsilon) {
         static_assert(std::is_floating_point<T>::value, "T must be a floating point type");
-        return abs(v) <= epsilon;
+        return v == 0.0 || abs(v) <= epsilon;
```

Comment:
This looks wrong, what is the idea here?

Reply:
On Appveyor, MSVC warns about possible division by 0 in some places where we use `vm::is_zero` to protect against exactly that. I suppose that the diagnostic is not smart enough to understand that our epsilon check implies that v != 0, so I'm making it explicit here. I'll add a comment to explain this.

Reply:
See b73b166.

Reply:
Ah that makes sense